.github/workflows/private-mirror-sync.yml

```diff
@@ -0,0 +1,22 @@
+name: Private Mirror Sync
```

Comment:
let's make sure it only runs on firebase/firebase-android-sdk repo.

Reply:
Done.